core/arch/x86/decode.c

```diff
@@ -741,6 +741,7 @@ read_evex(byte *pc, decode_info_t *di, byte instr_byte,
             return pc;
         }
         *is_evex = true;
+        SYSLOG_INTERNAL_WARNING_ONCE(MSG_AVX_512_SUPPORT_INCOMPLETE_STRING " @" PFX, pc);
```

Comment:
This is debug-build-only: we want a release-build notice.  SYSLOG takes the name w/o the MSG and no _STRING.  Args are via events.mc specifiers.

Reply:
Updated. I looked at the gen_events script. Seemed like it works like this (see update).
In terms of release build, how do I get a release build notice. Doesn't syslog go into the log, and log exists only in debug?

Reply:
SYSLOG goes *everywhere*, as specified by {stderr,msgbox,syslog}_mask (and always to log file).  So to show up in release it has to be ERROR not WARNING as you can see from the defaults.

Reply:
Done.